root/static/scripts/guess-case/modes.js

```diff
@@ -282,3 +291,27 @@ exports.Sentence = assign({}, DefaultMode, {
     {url: {href: 'https://musicbrainz.org/doc/Style/Language', target: '_blank'}}
   ),
 });
+
+exports.Turkish = assign({}, DefaultMode, {
+  description: l(
+    'This mode capitalises almost all words, the connectives ' +
+    'and the question suffixes used in a title are lowercased.'+
+    'In Turkish, capital of \'i\' is \'İ\' and of \'ı\' is \'I\' and'+
+    'this mode handles these. Some words, often proper nouns, ' +
+    'may need to be manually fixed according to the {url|Turkish ' +
+    'language guidelines}.',
```

Comment:
This needs language review from a native speaker. I’m heavily doubtful of the use of _capital_ here; also, the first sentence is misleading: You first read “This mode capitalises almost all words, the connectives and the question suffixes used in a title” and it sounds like a list of three items; only then you stumble over the “are lowercased” and see that you need to reparse the sentence.